src/styles/style.css

```diff
@@ -0,0 +1,87 @@
+@import url("../styles/variables.css");
+@import url("../styles/reset.css");
+@import url("../styles/stars.css");
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--black-text);
+}
+
+.product-card {
```

Comment:
Block styles should be in a separate file

Reply:
I have to move html tag into normalize.css?